config/kubernetes/demo/deployment.yaml

```diff
@@ -0,0 +1,176 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: peoplefinder
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  selector:
+    matchLabels:
+      app: peoplefinder
+  template:
+    metadata:
+      labels:
+        app: peoplefinder
+    spec:
+      containers:
+        - name: webapp
+          imagePullPolicy: Always
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/peoplefinder/peoplefinder-ecr:latest
+          ports:
+            - containerPort: 3000
+          command: ["./run.sh"]
+          env:
+            - name: ENV
+              value: 'dev'
```

Comment:
Should be "demo" 

On Peoplefinder, it had
ENV='<env name>' # either demo, staging, dev, production as appropriate
RACK_ENV='production'
RAILS_ENV='production'